src/v0/destinations/af/deleteUsers.js

```diff
@@ -60,87 +60,90 @@ const userDeletionHandler = async (userAttributes, config) => {
     body.status_callback_urls = filteredStatusCallbackUrlsArray;
   }
   const endpoint = `https://hq1.appsflyer.com/gdpr/opengdpr_requests?api_token=${config.apiToken}`;
-  for (let i = 0; i < userAttributes.length; i += 1) {
-    const userAttributeKeys = Object.keys(userAttributes[i]);
+  await Promise.all(
+    userAttributes.map(async ua => {
+      const userAttributeKeys = Object.keys(ua);
```

Comment:
do we need userAttributeKeys? we can use `ua. appsflyer_id` right in if conditions right?

Reply:
This will be addressed later.
https://www.notion.so/rudderstacks/User-deletion-feature-Code-review-comments-44c38e3996ee4b859da2a6188297f926